posh-svn.psd1

```diff
@@ -0,0 +1,56 @@
+@{
+
+# Script module or binary module file associated with this manifest.
+ModuleToProcess = 'posh-svn.psm1'
+
+# Version number of this module.
+ModuleVersion = '0.7.301'
+
+# ID used to uniquely identify this module
+GUID = 'f18820b6-2e02-41b4-afc5-de886bb1b848'
+
+# Author of this module
+Author = 'Chris Donnelly, Jeremy Skinner and contributors'
```

Comment:
We'd appreciate being included as the primary author, but fine with the rest.

Reply:
Yup.  Sorry - I was forking at one point instead of merging back in (hence all my changes to include my personal modules).  This was one of the changes.

I feel bad about this one, really shouldn't have put my name first.

Reply:
Could also just leave my name off too and be one of the contributors.